test/schema/artist/index.js

```diff
@@ -303,7 +368,47 @@ describe('Artist type', () => {
       const query = `
         {
           artist(id: "foo-bar") {
-            biography_blurb {
+            biography_blurb(partner_bio: true) {
+              text
+              credit
+            }
+          }
+        }
+      `;
+
+      return runQuery(query)
+        .then(data => {
+          expect(data).to.eql({
+            artist: {
+              biography_blurb: {
+                text: 'new catty bio',
+                credit: 'Submitted by Catty Partner',
+              },
+            },
+          });
+        });
+    });
+
+    it('returns the featured bio if it is for the requested partner', () => {
+      Artist.__ResetDependency__('gravity');
+      const gravity = sinon.stub();
+      Artist.__Rewire__('gravity', gravity);
```

Comment:
You can usually avoid this by moving the test code that was configuring the stub before into its own nested context, making `gravity` a blank slate stub in both contexts where they can be configured in their own `beforeEach` blocks.